packages/@ember/debug/index.ts

```diff
@@ -21,9 +21,7 @@ export type DebugFunctionType =
   | 'runInDebug'
   | 'deprecateFunc';
 
-export type AssertFunc =
-  | ((desc: string) => never)
```

Comment:
>   | ((desc: string) => never)

What was this overload even for? Seems weird...

Reply:
I have no idea, but it was messing with the `asserts` from TS so I removed, and everything seems to work 🤷